bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/profiles/SystemProfileFactory.java

```diff
@@ -114,4 +128,13 @@ public ProfileTypeUID getSuggestedProfileTypeUID(Channel channel, @Nullable Stri
         return SUPPORTED_PROFILE_TYPE_UIDS;
     }
 
+    @Reference(cardinality = ReferenceCardinality.MANDATORY, policy = ReferencePolicy.DYNAMIC)
```

Comment:
mandatory+dynamic does not really bring much value. Why not merely do `@Reference` here?